File: test/coop/rchain/lang/parser/RholangParserTest.java
package coop.rchain.lang.parser;

import com.intellij.lang.ParserDefinition;
import com.intellij.testFramework.ParsingTestCase;
import coop.rchain.RholangTestUtil;
import coop.rchain.lang.RhoParserDefinition;
import org.jetbrains.annotations.NotNull;

import java.nio.file.Paths;

public class RholangParserTest extends ParsingTestCase {
  public RholangParserTest() {
    super("", "rho", new RhoParserDefinition());
  }

  @Override
  protected String getTestDataPath() {
    return Paths.get(RholangTestUtil.getBaseTestDataPath(), "parser").toString();
  }

  public void testtoken(){
    doTest(true);
  }
}

File: test/coop/rchain/lang/parser/RholangParserTest.kt
package coop.rchain.lang.parser

import com.intellij.testFramework.ParsingTestCase
import coop.rchain.RholangTestUtil
import coop.rchain.lang.RhoParserDefinition
import java.nio.file.Paths

class RholangParserTest : ParsingTestCase("", "rho", RhoParserDefinition()) {

  override fun getTestDataPath(): String {
    return Paths.get(RholangTestUtil.baseTestDataPath, "parser").toString()
  }

  fun testtoken() {
    doTest(true)
  }
}
